### DISCUSSION TITLE
Wiki Documentation about ebusd messages and MQTT configuration

### DISCUSSION THREAD

**Discussion by esa1966** [john30/ebusd#1421] [Ideas]
Dear,
Thank a lot for the great job and the very detailed documentation ! 

**About the Wiki and ebusd configuration**
---------------------------------------------
I was able to have the connection in 1 day easily, but I lose a bit time concerning the step about the [Configuration](https://github.com/john30/ebusd/wiki/4.-Configuration) which have a lot of information's.
I was stuck at this point because I though it was necessary to select the right file before starting the ebusd daemon and this is not easy.

But, you make such a great job, that you it is no more necessary to do it (but maybe useful to read in a second step) : [Automatic-configuration)](https://github.com/john30/ebusd/wiki/4.7.-Automatic-configuration).

To improve the documentation, I think you can refer to chapter 4.7 on the main page (https://github.com/john30/ebusd/wiki) just after the title configuration.

Once configuration files are uploaded, I suppose it will keep the configuration and I don't need anymore the `--scanconfig` option in the `/etc/default/ebusd `. To speed up time and avoid issue once configured, can I remove this option in EBUSD_OPTS?

**About MQTT and HA**
---------------------------
I have chosen to install the daemon on a debian VM. 

**It is working but it is not clear on some points :** 

ebus adapter also refer to a MQTT, but only for some point of the board itself. Maybe a explanation can be given about this configuration and values (it is not the MQTT for all value in the boiler).
Another user had the same [confusion](https://community.home-assistant.io/t/problems-using-ebus-v5-ebusd/781043)

I configured on the web page of the adapter "mqtt:://USER:PASSWORD@IPHomeAssistant/eas", but I don't see something in HA. 
But in MQTT explorer I see information's. Maybe it is not correct.

**- Concerning MQTT for ebusd, I have following questions :** 
At this point, it is working with the command ebusd : 
`ebusd --mqtthost 192.168.1.111 --mqttport 1883 --mqttpass=PASSWORD --mqttuser=MQTTUSER --mqttint=/etc/ebusd/mqtt-hassio.cfg --mqttjson
`
If I want to use the daemon, and 'systemctl start ebusd', how I can set the above configuration. I suppose I have to configure the file in `/etc/default/ebusd `but there is no remark about such configuration.
Should I add some line for each option ? Can you advise me about this point ?

**Remark :** 
In the installation process, I get : `Make the daemon autostart with 'systemctl enable ebusd`'
In the Wiki [Run](https://github.com/john30/ebusd/wiki/2.-Run), it refer also to : `sudo chkconfig ebusd on`
What is it used for  ?

About missing values from boiler 
----------------------------------
I can acquire some info from the boiler which are missing in HA through MQTT `ebusctl read -f bai WaterPressure`
How can I add this variable for MQTT ?
I also want to scan the status01 more often (each 5 seconds) because this is the most important variable (for me) which also contains the status of the boiler. Is it possible ?
remark : I already put the POLL scan to 1 second, but it take to many time to scan all values.

Thank a lot for your support
Eric

**Comment by troffasky** [comment]
systemctl or chkconfig? The latter is for Red Hat-derived systems, I believe. So you run one or the other, not both.

**Comment by esa1966** [comment]
Hi,

**To start the service**
I run on Debian. So to start the daemon : systemctl start ebusd
About the configuration, I find that I need to add the setting at the end of the OPTS.  In file /etc/default/ebusd : 
`EBUSD_OPTS="--scanconfig -d ens:192.168.1.201:9999 --mqtthost 192.168.1.111 --mqttport 1883 --mqttpass=PASSWORD --mqttuser=MQTTUSER --mqttint=/etc/ebusd/mqtt-hassio.cfg --mqttjson --pollinterval=1"
`
**About to speed up the communication and just take variable I'm interested**
I need to filter the "variable" in file /etc/ebusd/mqtt-hassio.cfg
I set the following : 
`filter-name = ^status01$|^FlowTemp$|^ReturnTemp$|^HwcTemp$|^ModulationTempDesired$|^PumpPower$|^WaterPressure$|^CirPump$|^DCRoomthermostat$|^FanSpeed$|^Flame$|^Fluegasvalve|^Ignitor$|HeatingSwitch$`


![image](https://github.com/user-attachments/assets/de19da34-9597-4255-84b7-e3d68ee1a701)
 
Now, I want to suppress some varaible which I don't use anymore in HA.
In HA, can I delete the device "ebusd bai" in Setting -> DEVICES -> MQTT ?
It will be rebuild automatically ?
If not, how can I suppress those variables ?

Thank for the support.
